tools/precommit.sh

```diff
@@ -57,38 +57,4 @@ echo -e "All targets were built successfully. Starting unit tests' run.\n"
 $MAKE unittests_run || exit 1
 echo -e "\nUnit tests completed successfully. Starting full testing.\n"
 
-RUN_IDS=""
-
-for TARGET in $TARGETS
-do
- ENGINE=$OUT_DIR/$TARGET/jerry
- LOGS_PATH_PARSE_ONLY=$OUT_DIR/$TARGET/check_parse_only
- LOGS_PATH_FULL=$OUT_DIR/$TARGET/check
-
- # Parse-only testing
- INDEX=0
- for TESTS_PATH in "./tests/benchmarks/jerry"
- do
-   ./tools/runners/run-precommit-check-for-target.sh $ENGINE $LOGS_PATH_PARSE_ONLY/$INDEX $TESTS_PATH --parse-only &
-   RUN_IDS="$RUN_IDS $!";
-   INDEX=$((INDEX + 1))
- done
-
- # Full testing
- INDEX=0
- for TESTS_PATH in "./tests/jerry" "./tests/jerry-test-suite/precommit_test_list"
- do
-   ./tools/runners/run-precommit-check-for-target.sh $ENGINE $LOGS_PATH_FULL/$INDEX $TESTS_PATH &
-   RUN_IDS="$RUN_IDS $!";
-   INDEX=$((INDEX + 1))
- done
-done
-
-RESULT_OK=1
-for RUN_ID in $RUN_IDS
-do
-  wait $RUN_ID || RESULT_OK=0
-done;
-[ $RESULT_OK -eq 1 ] || exit 1
-
-echo -e "Full testing completed successfully\n\n================\n\n"
+./tools/precommit-full-testing.sh ${OUT_DIR} ${TARGETS} || exit 1
```

Comment:
Wouldn't this pass the `${TARGETS}` as the 2nd, 3rd and etc args? For example if the `TARGETS` are: debug.linux release.linux. the command will look like this:
` ./tools/precommit-full-testing.sh ${OUT_DIR} debug.linux release.linux `

Maybe we need to enclose the arguments into quotes.